apps/backend/src/libs/modules/http/http.ts

```diff
@@ -1,3 +1,3 @@
 export { HTTPCode } from "./libs/enums/enums.js";
-export { HTTPError } from "./libs/exceptions/exceptions.js";
+export { AuthError, HTTPError } from "./libs/exceptions/exceptions.js";
```

Comment:
AuthError should be exported from ~/modules/auth/auth.ts

Reply:
I currently have `AuthError` defined in the shared package  in exemption folder where all errors are managed. 

Should I just move the import to the `auth` module, or do you want the entire `AuthError` class to be relocated and managed within the `auth` module itself?

Reply:
The class itself should be in shared package. You need to move the import to the `auth` module